integrations/fake-integration/fake_org_data/generator.py

```diff
@@ -8,33 +9,51 @@
 
 fake = Faker()
 
+DEFAULT_ENTITIES_AMOUNT = 400
+DEFAULT_ENTITY_KB_SIZE = 1  # Will be multiplied by 1k (approx size of entity json)
+DEFAULT_LATENCY_FACTOR = (
+    0  # Will be multiplied by a random number of factor * 5 if larger than zero
+)
 
-def generate_fake_persons(
-    department_id: Union[str, None] = None, amount: Union[int, None] = None
+
+async def generate_fake_persons(
+    department_id: Union[str, None],
+    amount: int,
+    entity_kb_size: int,
+    latency: int,
 ) -> Dict[str, Any]:
```

Comment:
lets add description for what are possible values and what each one is used for

Reply:
Added